Update getting-started-with-swashbuckle.md (#31647)
dotnet/AspNetCore.Docs: aspnetcore/tutorials/getting-started-with-swashbuckle.md

author: zuckerthoben
description: Learn how to add Swashbuckle to your ASP.NET Core web API project to integrate the Swagger UI.
ms.author: scaddie

> monikerRange: '>= aspnetcore-3.1'

Author: @captainsafia this change removes redirect to .NET 7

ms.custom: mvc
ms.date: 11/09/2021
uid: tutorials/get-started-with-swashbuckle
---
# Get started with Swashbuckle and ASP.NET Core

> > [!NOTE]

Reviewer: Should this note be configured to only display for net8.0 and later?

Author: > Should this note be configured to only display for net8.0 and later?

Fixed

> > Build-time OpenAPI document generation with Swashbuckle is not supported in .NET 8 and later. For a supported  build-time  alternative, see <xref:tutorials/web-api-help-pages-using-swagger>. Run-time support is not impacted.

Author: @captainsafia The original was off putting. I'm hoping to tone it down, closer to the actual limitations.

Reviewer: This change looks good!

> > Build-time OpenAPI document generation with Swashbuckle is not supported in .NET 8 and later. For a supported  build-time  alternative, see <xref:tutorials/web-api-help-pages-using-swagger>. Run-time support is not impacted.

Reviewer 2: ```suggestion
> Build-time OpenAPI document generation with Swashbuckle isn't supported in .NET 8 and later. For a supported build-time alternative, see <xref:tutorials/web-api-help-pages-using-swagger>. Swashbuckle still supports run-time document generation.
```

Reviewer: > Swashbuckle still supports run-time document generation.

The phrasing here feels off to me. Maybe "Run-time document generation is still supported in .NET 8."
